families/families.js

```diff
@@ -58,14 +50,13 @@ async function displayFamilies() {
             });
```

Comment:
Here is where we could display the name of the bunny after creating an element. Then the element would equal bunny.(name).